lrlex/src/lib/parser.rs

```diff
@@ -351,11 +351,19 @@ where
                     &mut grmtools_section_span_map,
                     &mut grmtools_section_lex_flags,
                 )?;
-                if i == self.src.len() {
-                    return Err(self.mk_error(LexErrorKind::PrematureEnd, i));
+                eprintln!("foo '{:?}'", &self.src[i..]);
```

Comment:
Should be removed.

Reply:
fixed in [57b94aa](https://github.com/softdevteam/grmtools/pull/542/commits/57b94aac5173e0220476172e445b3c844586fc4f)